clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -3271,6 +3268,16 @@ private long getLatestLeaderOffsetAndHybridTopicOffset(
         : cachedPubSubMetadataGetter
             .getOffset(getTopicManager(sourceRealTimeTopicKafkaURL), leaderTopic, partitionToGetLatestOffsetFor);
 
+    if (lastOffsetInRealTimeTopic < 0) {
+      if (!REDUNDANT_LOGGING_FILTER.isRedundantException("Got a negative lastOffsetInRealTimeTopic")) {
+        LOGGER.warn(
+            "Unexpected! Got a negative lastOffsetInRealTimeTopic ({})! Will return Long.MAX_VALUE as the lag.",
```

Comment:
can we also print the value of Long.MAX_VALUE for easier comparison from the log.

Reply:
Sure.